packages/cli/src/executions/executions.service.ee.ts

```diff
@@ -10,4 +14,21 @@ export class EEExecutionsService extends ExecutionsService {
 		// Get all workflows
 		return getSharedWorkflowIds(user);
 	}
+
+	static async getExecution(
+		req: ExecutionRequest.Get,
+	): Promise<IExecutionResponse | IExecutionFlattedResponse | undefined> {
+		const execution = await super.getExecution(req);
+
+		if (!execution) return;
+
+		const workflow = Object.assign(new WorkflowEntity(), execution.workflowData);
```

Comment:
can we replace this `Object.assign` with `repository.create` for better type-safety?